### PR TITLE
Update person display UN SDG generated discovery links

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by rmathew1011 — APPROVED** [review]

**Review by kaladay — COMMENTED** [review]
I am a bit confused with this one.

Which issue is this solving?
I see this issue linked:
- https://github.com/TAMULib/scholars-discovery/issues/279

I see this issue referenced in the comment:
- https://github.com/TAMULib/scholars-discovery/issues/211

**Comment by ghost** [comment]
@kaladay this PR is for https://github.com/TAMULib/scholars-discovery/issues/279. I mentioned issue https://github.com/TAMULib/scholars-discovery/issues/211 in the description because I am not happy with making changes like these in addition to not liking URLs with excessive query parameters in a template. For an administrative interface to manage these templates would not easily be able to craft the URLs. I am thinking of a key-value pair to resolve them based on some opaque identifier. Or some other solution. Either way that is way out of scope to just update the links in the templates.

**Comment by kaladay** [comment]
Please separate the white space `+` vs ` ` vs `%20` changes to a different PR for lines unrelating to the issue.

I'm looking for a difference so that I can understand the changed and after some formatting I was able to see the following:

```diff
--- /tmp/one	2023-05-24 13:35:43.997392450 -0500
+++ /tmp/two	2023-05-24 13:35:49.027392433 -0500
@@ -1,4 +1,4 @@
-      <a class="m-1" href="discovery/Scholarly+Works?collection=individual&
+      <a class="m-1" href="discovery/Scholarly%20Works?collection=individual&
 fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&
 facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&
 type.type=STRING&
@@ -47,15 +47,18 @@ citationCount.rangeStart=0&
 citationCount.rangeEnd=20000&
 citationCount.rangeGap=100&
 citationCount.sort=INDEX,DESC&
-class.filter=Document+OR+type:creativeWork&
+class.filter=Document%20OR%20type:creativeWork&
 class.opKey=EXPRESSION&
-tags.filter={{this}}&
+tags.filter=4%20Quality%20Education&
 tags.opKey=EQUALS&
-filters=class,tags&
+authors.filter=Craig,%20Cheryl&
+authors.opKey=EQUALS&
+filters=class,tags,authors&
 boost=title,2&
 boost=keywords,2&
 boost=abstract,2&
 sort=score,DESC&
 sort=title_sort,ASC&
 hl=title&
-hl=bookTitle" target="_blank">
+hl=bookTitle&
+expanded=Author,UN%2520Sustainable%2520Development%2520Goals" target="_blank">
```

I am seeing specifically named authors added, such as `Craig`.
I am also seeing the filter changed from `{{this}}` top `4%20Quality%20Education`

These don't seem correct to me.
Can you explain why these specific names are needed as it is unclear to me?
(Also, those screenshots attached to the issued are too small/blurry to read anything.)

**Comment by ghost** [comment]
@kaladay I will double check the URLs and test again.

There are two types of URLs in these templates: one navigating the UI via Angular routing, and the other direct request to the scholars-discovery API. The URL changes replacing `+` with ` ` is in the Angular path parameter for routing. The query parameters to the scholars-discovery API should accept any ` `, `+`, or `%20`.

File: src/main/resources/defaults/directoryViews/research/default.html [hunk 1]
```diff
@@ -60,7 +60,7 @@
...
...hors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle">
```
Comment (ghost) [inline src/main/resources/defaults/directoryViews/research/default.html]:
```
---discovery/Scholarly+Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document+OR+type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
+++discovery/Scholarly Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
```

File: src/main/resources/defaults/discoveryViews/publications/default.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
...
...hors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle">
```
Comment (ghost) [inline src/main/resources/defaults/discoveryViews/publications/default.html]:
```
---discovery/Scholarly+Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document+OR+type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
+++discovery/Scholarly Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
```

File: src/main/resources/defaults/displayViews/documents/other/tags.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
...&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle" target="_blank">
```
Comment (ghost) [inline src/main/resources/defaults/displayViews/documents/other/tags.html]:
```
---discovery/Scholarly+Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document+OR+type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
+++discovery/Scholarly Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
```

File: src/main/resources/defaults/displayViews/persons/asideTemplate.html [hunk 4]
```diff
@@ -20,7 +20,7 @@
...
...&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle" target="_blank">
```
Comment (ghost) [inline src/main/resources/defaults/displayViews/persons/asideTemplate.html]:
```
---discovery/Scholarly+Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document+OR+type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
+++discovery/Scholarly Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document OR type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
```

File: src/main/resources/defaults/displayViews/persons/asideTemplate.html [hunk 5]
```diff
@@ -118,7 +118,7 @@
...
...authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document%20OR%20type:creativeWork&class.opKey=EXPRESSION&tags.filter=4%20Quality%20Education&tags.opKey=EQUALS&authors.filter=Craig,%20Cheryl&authors.opKey=EQUALS&filters=class,tags,authors&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle&expanded=Author,UN%2520Sustainable%2520Development%2520Goals" target="_blank">
```
Comment (ghost) [inline src/main/resources/defaults/displayViews/persons/asideTemplate.html]: Missed one

```
---discovery/Scholarly+Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document+OR+type:creativeWork&class.opKey=EXPRESSION&tags.filter={{this}}&tags.opKey=EQUALS&filters=class,tags&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle
+++discovery/Scholarly%20Works?collection=individual&fl=type,title,bookTitle,publicationDate,publisher,authors,authorOrganization,doi,pmcid,uri,tags&facets=type,publicationDate,authors,authorOrganization,publicationVenue,publisher,keywords,tags,altmetricScore,citationCount&type.type=STRING&type.pageSize=10&type.pageNumber=1&type.sort=COUNT,DESC&publicationDate.type=DATE_YEAR&publicationDate.pageSize=10&publicationDate.pageNumber=1&publicationDate.sort=INDEX,DESC&authors.type=STRING&authors.pageSize=10&authors.pageNumber=1&authors.sort=COUNT,DESC&authorOrganization.type=STRING&authorOrganization.pageSize=10&authorOrganization.pageNumber=1&authorOrganization.sort=COUNT,DESC&publicationVenue.type=STRING&publicationVenue.pageSize=10&publicationVenue.pageNumber=1&publicationVenue.sort=COUNT,DESC&publisher.type=STRING&publisher.pageSize=10&publisher.pageNumber=1&publisher.sort=COUNT,DESC&keywords.type=STRING&keywords.pageSize=10&keywords.pageNumber=1&keywords.sort=COUNT,DESC&tags.type=STRING&tags.pageSize=10&tags.pageNumber=1&tags.sort=COUNT,DESC&altmetricScore.type=NUMBER_RANGE&altmetricScore.pageSize=10&altmetricScore.pageNumber=1&altmetricScore.rangeStart=0&altmetricScore.rangeEnd=2000&altmetricScore.rangeGap=100&altmetricScore.sort=INDEX,DESC&citationCount.type=NUMBER_RANGE&citationCount.pageSize=10&citationCount.pageNumber=1&citationCount.rangeStart=0&citationCount.rangeEnd=20000&citationCount.rangeGap=100&citationCount.sort=INDEX,DESC&class.filter=Document%20OR%20type:creativeWork&class.opKey=EXPRESSION&tags.filter=4%20Quality%20Education&tags.opKey=EQUALS&authors.filter=Craig,%20Cheryl&authors.opKey=EQUALS&filters=class,tags,authors&boost=title,2&boost=keywords,2&boost=abstract,2&sort=score,DESC&sort=title_sort,ASC&hl=title&hl=bookTitle&expanded=Author,UN%2520Sustainable%2520Development%2520Goals
```

**Comment by ghost** [comment]
@kaladay I am missing the author filter. Adding it back. I will roll back and add separate commits updating URL format from templating filters.

**Pull request closed without merging**